### PR TITLE
:pencil: add Chinese translation

### PR DESCRIPTION
**Pull request by JalenChuh**
add Chinese translation

This project is so great, thanks for your contribution! :bow:

### CODE REVIEW

File: readme_cn.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+GitHub extra pins allow you to pin more than 6 repositories in your profile using a GitHub readme profile.
+
+Yey! You are no longer limited to 6 pinned repositories.
+
+### Usage
+
+Copy-paste this code into your readme and change the links.
+
+Endpoint: `api/pin?username=anuraghazra&repo=github-readme-stats`
+
```
Comment (anuraghazra) [inline readme_cn.md:123]: LGTM, You missed this part i think.

**Comment by anuraghazra** [comment]
This is awesome @JalenChuh Thanks!

File: readme_cn.md (hunk 1 above)
Comment (JalenChuh) [reply readme_cn.md:123]: So sorry, I forgot to explain for it.

I have some puzzle in this part, maybe I know the meaning, but I can't translate it exactly.
So I choose not to translate it.

If you think it's necessary to do that, I will try it. But I think that keep as it is is the better.

Comment (anuraghazra) [reply readme_cn.md:123]: I see, no problem then, LGTM! Thanks!

**Pull request merged**